internal/contour/listener.go

```diff
@@ -75,8 +75,13 @@ type ListenerVisitorConfig struct {
 	// If not set, defaults to false.
 	UseProxyProto bool
 
-	// MinimumProtocolVersion defines the min tls protocol version to be used
-	MinimumProtocolVersion envoy_api_v2_auth.TlsParameters_TlsProtocol
+	// MinimumTLSVersion defines the minimum TLS protocol version the proxy should accept.
+	MinimumTLSVersion envoy_api_v2_auth.TlsParameters_TlsProtocol
+
+	// DefaultHTTPVersions defines the default set of HTTPS
```

Comment:
This field applies to both TLS and non-TLS listeners, right? The comment is misleading that it only applied to TLS ones.

Reply:
Yeh it's applied to both, but since we don't support h2c, it's really a moot point. I'll attempt to clarify this comment.

Reply:
Updated the comment.